pkg/controller/certapi/certificate.go

```diff
@@ -36,11 +37,18 @@ func (c *Controller) HandleCertificate() http.Handler {
 	return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
 		ctx := observability.WithBuildInfo(r.Context())
 
+		var blame = controller.BlameNone
+		var result = controller.APIResultOK()
+
+		defer stats.RecordWithTags(ctx, []tag.Mutator{blame, result}, mRequest.M(1))
```

Comment:
@icco Do you know if the linter (complain about blame/result variables not used later in the code) is correct?

I'm a bit fuzzy here, would this defer work as intended (use the blame/result variable's value right before return), or will it capture their current value and thus the blame/result variables later in the file is ignored?

Reply:
I'm pretty sure it's the latter.

Reply:
Fixed by capturing a pointer in the closure. Linter no longer complain about unused variables.